<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UOAppointmentsTableDialog</class>
 <widget class="QDialog" name="UOAppointmentsTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сервис Управления потоками пациентов (очередями)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Врач:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="lvDoctors"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Доступные талоны:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tblAppointments">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string>Загрузка списка талонов...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRegisterReferral">
       <property name="text">
        <string>Зарегистрировать направление</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpdateList">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSetAppointment">
       <property name="text">
        <string>Записать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
